frontend/src/components/Bundle/Bundle.js

```diff
@@ -131,8 +132,8 @@ class Bundle extends React.Component<
             .then(callback)
             .catch(errorHandler);
 
-        callback = (storeInfo) => {
-            if (!storeInfo || storeInfo.length == 0) return;
+        callback = ({ data: storeInfo }) => {
```

Comment:
Do we need to modify the `cl bundle ls` command to properly handle the new output structure as well?

Reply:
Do you mean `cl store ls`? I tested it's working fine.